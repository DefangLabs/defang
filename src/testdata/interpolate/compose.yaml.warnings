 ! Environment variable "NODE_ENV" is not used; add it to `.env` if needed
 ! Environment variable "NODE_ENV" is not used; add it to `.env` or it may be resolved from config during deployment
 ! can only have more than one replica for PRO tier and PRODUCTION deployment mode. Will set replicas to 1 for service(s) [interpolate]
 ! service "interpolate": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors
missing configs ["NODE_ENV" "POSTGRES_PASSWORD" "def"] (https://docs.defang.io/docs/concepts/configuration)