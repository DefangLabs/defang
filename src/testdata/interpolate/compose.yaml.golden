name: interpolate
services:
  interpolate:
    environment:
      BRACED: interpolate
      DB: postgres://user:${POSTGRES_PASSWORD}@db:5432/db
      NAME: interpolate
      NODE_ENV: ${NODE_ENV}
      NOP: abc$def
      NOP_BRACED: abc${def}
      PORT: "8080"
      interpolate: value
    image: alpine
    networks:
      default: null
networks:
  default:
    name: interpolate_default
